.travis.yml

```diff
@@ -2,6 +2,11 @@ language: php
 
 php: [5.3, 5.4, 5.5, 5.6, hhvm]
 
+matrix:
+  include:
+    - php: 5.6
+      env: ZOMBIE_VERSION='@~2.0'
```

Comment:
In fact we can run Zombie 2.x version with single PHP version to be sure that it's working and use latest Zombie version for all PHP versions.